### DISCUSSION TITLE
blog/react_compiler_1/

### DISCUSSION THREAD

**Discussion by giscus** [yongsk0066/yongsk0066.github.io#6] [Comments]
# blog/react_compiler_1/

React Compiler에 대해 깊이 파헤쳐보고자 한다. 우선은 바벨 플러그인을 통해 컴파일러의 진입점을 살펴보자.

https://yongsk0066.github.io/blog/react_compiler_1/

**Comment by ProgWon** [comment]
재밌네요! 좋은 글 감사합니다!

**Comment by wonkyDD** [comment]
한국의 dan이 여기 있었군요.. 존경합니다 GOAT
